app/models/commons_license.rb

```diff
@@ -9,7 +9,7 @@ class CommonsLicense < ActiveRecord::Base
   DeedFormat  =   "http://#{Site}/licenses/%{code}/%{version}/"
   LegalFormat =   "http://#{Site}/licenses/%{code}/%{version}/legalcode"
 
-  default_scope :order => 'number ASC'
+  default_scope { order('number ASC') }
```

Comment:
Missing `->` here?

Reply:
I see other `default_scope` are missing `->`. I'm guessing this is intentional. :smile:

Reply:
default scopes do not need them. They are implied I believe.